budget/Budget.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Schema.Types.ObjectId;
```

Comment:
Not sure you are going to need this line on this specific file, since nothing is being linked in here.